modules/deploy/pages/deployment-option/cloud/aws-privatelink.adoc

```diff
@@ -0,0 +1,393 @@
+
+Consider using the PrivateLink endpoint service if you have multiple VPCs and could benefit from a more simplified approach to network management:
+
+* PrivateLink allows overlapping CIDR ranges in VPC networks.
+* PrivateLink does not limit the number of VPC connections. VPC peering on the other hand, is limited to 125 connections (see https://aws.amazon.com/privatelink/faqs/["How scalable is AWS PrivateLink?"]).
+
+After <<get-an-access-token,getting an access token>>, you can <<enable-privatelink-endpoint-service-for-new-clusters,enable PrivateLink when creating a new cluster>>, or you can <<enable-privatelink-endpoint-service-for-existing-clusters,enable PrivateLink for existing clusters>>.
+
+TIP: Make sure to replace the variable values in the code examples on this page with your own values, before running the commands in the terminal or in a script.
+
+== Requirements
+
+* Install `rpk`.
+* Your Redpanda cluster and <<create-client-vpc,VPC>> must be in the same region.
+* You must use the Redpanda Cloud API to enable the Redpanda endpoint service for your clusters. Follow the steps below to <<get-an-access-token,get an access token>>.  
+* Use the AWS CLI to create a new client VPC or modify an existing one to use the PrivateLink endpoint.
+
+== Get an access token 
+
+. Specify the base URL of the Redpanda Cloud API:
++
+[,bash]
+----
+PUBLIC_API_ENDPOINT="https://api.cloud.redpanda.com"
+----
+
+. In the Redpanda Cloud UI, go to **Namespaces** and select the namespace where you want to create a cluster.
++
+Copy the namespace ID (UUID) from the URL in the browser.
++
+[,bash]
+----
+NAMESPACE_ID=<uuid>
+----
+
+. Go to the organization (org) level **Home** in the UI, and navigate to **Clients**. If you don't have an existing client, you can create a new one by clicking **Add client**.
++
+Copy the client ID and secret.
++
+[,bash]
+----
+CLOUD_CLIENT_ID=<client-ID>
+CLOUD_CLIENT_SECRET=<client-secret>
+----
+
+. Get an API token using the client ID and secret. You can click the **Request an API token** link to see code examples to generate the token.
++
+[,bash]
+----
+AUTH_TOKEN=`curl -s --request POST \
+    --url 'https://auth.prd.cloud.redpanda.com/oauth/token' \
+    --header 'content-type: application/x-www-form-urlencoded' \
+    --data grant_type=client_credentials \
+    --data client_id=$CLOUD_CLIENT_ID \
+    --data client_secret=$CLOUD_CLIENT_SECRET \
+    --data audience=cloudv2-production.redpanda.cloud | jq .access_token | sed 's/"//g'`
+----
+
+== Enable PrivateLink endpoint service for new clusters
+
+. Call https://docs.api.redpanda.com/#post-/v1beta1/networks[`POST /v1beta1/networks`] to create a network.
++
+Make sure to supply your own values in the example request below. Store the network ID (`network_id`) after the network is created.
```

Comment:
Will users need to specify the network ID in a later step/task? If so, perhaps mention what that is so they understand this is important and not just a suggestion.